addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/discovery/AVMFritzDiscoveryService.java

```diff
@@ -113,9 +110,10 @@ public void onDeviceAddedInternal(DeviceModel device) {
             final ThingUID bridgeUID = bridgeHandler.getThing().getUID();
             final Map<String, Object> properties = new HashMap<>();
             properties.put(THING_AIN, device.getIdentifier());
-            properties.put(Thing.PROPERTY_VENDOR, device.getManufacturer());
-            properties.put(Thing.PROPERTY_SERIAL_NUMBER, device.getIdentifier());
-            properties.put(Thing.PROPERTY_FIRMWARE_VERSION, device.getFirmwareVersion());
+            properties.put(PROPERTY_VENDOR, device.getManufacturer());
+            properties.put(PROPERTY_MODEL_ID, device.getDeviceId());
+            properties.put(PROPERTY_SERIAL_NUMBER, device.getIdentifier());
+            properties.put(PROPERTY_FIRMWARE_VERSION, device.getFirmwareVersion());
 
             final DiscoveryResult discoveryResult = DiscoveryResultBuilder.create(thingUID).withProperties(properties)
                     .withRepresentationProperty(device.getIdentifier()).withBridge(bridgeUID)
```

Comment:
@martinvw I have a question:  When using `withRepresentationProperty()` method, what do I have to pass as parameter?  The `name` of the property or the `value` for the representational property?

Reply:
Good catch, see https://github.com/eclipse/smarthome/blob/master/bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/DiscoveryResult.java#L77

Reply:
Thanks. I fixed it.